publish/publish.proj

```diff
@@ -157,10 +157,11 @@
     </PropertyGroup>
     <MakeDir Directories="$(DownloadDirectory)"
              Condition="!Exists('$(DownloadDirectory)')" />
-    <DownloadBlobsFromAzure AccountName="$(AzureAccountName)"
+    <DownloadFromAzure AccountName="$(AzureAccountName)"
                            AccountKey="$(AzureAccessToken)"
                            ContainerName="$(ContainerName)"
                            BlobNames="@(_CoreHostPackages)"
+                           BlobNamePrefix="Runtime/$(SharedFrameworkNugetVersion)"
```

Comment:
In the DownloadFromAzure task, if you specify the BlobNamePrefix, it will trim that path from the directory it downloads to locally - https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs#L99

So, by specifying the prefix we get local paths like `Bin\ForPublishing` instead of `Bin\ForPublishing\Runtime\blah`